config/brakeman.ignore

```diff
@@ -180,6 +242,37 @@
       "confidence": "Medium",
       "note": ""
     },
+    {
+      "warning_type": "Cross-Site Scripting",
+      "warning_code": 2,
+      "fingerprint": "e4c4ac3c935ca5fb6cfd06500134811926beab214e5b379d800e1b6d64ab7f4b",
+      "check_name": "CrossSiteScripting",
+      "message": "Unescaped model attribute",
+      "file": "app/views/cases/letters/show.html.slim",
+      "line": 27,
+      "link": "https://brakemanscanner.org/docs/warning_types/cross_site_scripting",
+      "code": "Letter.new(letter_params[:letter_template_id], Case::Base.find(params[:case_id]).decorate).letter_address",
+      "render_path": [
+        {
+          "type": "controller",
+          "class": "Cases::LettersController",
+          "method": "show",
+          "line": 15,
+          "file": "app/controllers/cases/letters_controller.rb",
+          "rendered": {
+            "name": "cases/letters/show",
+            "file": "app/views/cases/letters/show.html.slim"
+          }
+        }
+      ],
+      "location": {
+        "type": "template",
+        "template": "cases/letters/show"
+      },
+      "user_input": null,
+      "confidence": "High",
+      "note": "Cloned object in Letter.rb with relevant escaped vals now passed to view"
+    },
```

Comment:
it seems like even we have a solution for escaping the value in cases, the above warning couldn't be eliminated? 🤔

Reply:
did you run it with `bundle exec brakeman`? It's not showing up for me when I run it locally